.gitignore

```diff
@@ -28,3 +28,8 @@ rspec.xml
 .vendor
 integration_run
 .mvn/
+qa/.ssh_config
+qa/.vagrant
+qa/.rspec
+qa/acceptance/.vagrant
+qa/Gemfile.lock
```

Comment:
We might want to add `aq/vendor` to this list, in case anyone is doing `bundler install --path vendor` like myself.